src/components/Home/index.scss

```diff
@@ -16,4 +16,5 @@
 
 .Home-EmulatorCard-Info {
   margin: 16px;
+  min-height: 214px; // fix wrapping text making cards different sizes
```

Comment:
Let's just make the title of each card 2 line-heights tall. This should remove the need to calculate here and also hopefully aligns better.

Reply:
I tried that and it was wonky on normal modes.. too much whitespace ...  for smaller screens, I went with ellipsis for now:

<img width="1043" alt="Screen Shot 2020-04-14 at 5 20 43 PM" src="https://user-images.githubusercontent.com/702990/79286056-478dc180-7e74-11ea-9ca7-f3cd6e40eb4e.png">

It was better than this:

<img width="1438" alt="Screen Shot 2020-04-14 at 5 21 51 PM" src="https://user-images.githubusercontent.com/702990/79286118-7015bb80-7e74-11ea-9243-4069266e5b39.png">


And note that it only exhibits this mode on a 13 in with devtools on right. so it's in between phone (2 col) and desktop, where 3 cols shows all text